Libraries/src/Amazon.Lambda.Annotations/LambdaGlobalPropertiesAttribute.cs

```diff
@@ -0,0 +1,24 @@
+﻿
+using System;
+
+namespace Amazon.Lambda.Annotations
+{
+    /// <summary>
+    /// Used when deploying Lambda functions as a .NET executable instead of a class library. Ensure the .NET project's output type is configured for `Console Application`. In the project file the `OutputType` property will be set to `Exe`.
```

Comment:
This is docs for the `GenerateMain` property. The docs here should be about how this attribute is for setting global properties that will affect the code and CloudFormation generation for all of the projects in the assembly.